crates/bevy_asset/src/asset_server.rs

```diff
@@ -389,6 +414,7 @@ impl AssetServer {
         asset_path.into()
     }
 
+    /// Loads one or more assets from the folder at the provided path.
```

Comment:
This makes it sounds like there will be a choice to load one or more. This methods will load all files with a matching asset loader. it will do so recursively, and will fail if given a path to a file